gpu4pyscf/lib/cupy_helper.py

```diff
@@ -339,250 +340,6 @@ def transpose_sum(a):
         raise RuntimeError('failed in transpose_sum kernel')
     return a
 
-LMAX_ON_GPU = 8
```

Comment:
The following functions are deleted?

Reply:
They are duplication of code in the same file